assets/extensions/examples/online-extensions.js

```diff
@@ -30,76 +32,73 @@ window.OPENSHIFT_CONSTANTS.HELP = {
   "default":                 "https://docs.openshift.com/online/latest/welcome/index.html"
 };
 
-/*
-  Custom angular module 
-*/
-angular.module('openshiftOnlineConsoleExtensions', ['openshiftConsole'])
-  .run(function(HawtioExtension) {
-
-    /*
-      Request system status from statuspage.io
-    */
-    var system_status_elem = $('<a href="http://status.openshift.com/" target="_blank" class="nav-item-iconic system-status project-action-btn" style="display: none;">');
-    var system_status_elem_mobile = $('<div row flex class="navbar-flex-btn system-status-mobile" style="display: none;">');
-
-    $.getJSON("https://m0sg3q4t415n.statuspage.io/api/v2/summary.json", function (data) {
-      var n = (data.incidents || [ ]).length;
-    
-      if (n > 0) {
-        var issueStr = n + ' open issue';
-        if (n !== 1) {
-          issueStr += "s";
+
+angular
+  // dont know if we really need openshiftOnlineConsoleExtensions,
+  // if the file exists, it will add the extensions, if it does not, nothing.
+  .module('openshiftConsole')
```

Comment:
don't change this, I specifically did it because this is an example we want other people to copy, and we don't want other people putting their extensions underneath our module, for one this is a guarantee of ordering, this modules config/run methods will always be called after the openshiftConsole's config/run methods